src/utils/app.constant.ts

```diff
@@ -57,3 +57,8 @@ export enum Numbers {
   ONE = 1,
   TEN = 10,
 }
+
+export enum Types {
```

Comment:
@upendraTekdi rename it to the CohortTypes

Reply:
done